### ISSUE TITLE
Vertical dividers in contextual footer should be full height

### ISSUE THREAD

**Issue by yaili** [canonical-websites/www.ubuntu.com#1684]
Reviewing /about but applies to all pages.

**Comment by barrymcgee** [comment]
This has been proposed to the Vanilla Brochure Theme so this issue should be unblocked and moved to review once this version of VBT has been included.

**Comment by yaili** [comment]
Should this not be fixed by using equal heights?

**Comment by barrymcgee** [comment]
@yaili Is there ever a situation where items separated from each other by the divider pattern should not be full height?

**Comment by yaili** [comment]
I don't know.

**Comment by barrymcgee** [comment]
@yaili I can't see where there would be which is why making them equal height should be inherent to this pattern and not require a utility class.

**Comment by yaili** [comment]
Are we making the contextual footer a separate pattern? Isn't it just the grid with equal heights and dividers? It seems that in ubuntu.com the orange strip is an hr, which sounds right. Can't we do that?

**Comment by barrymcgee** [comment]
No, yes and Vanilla `<hr>` pattern is 1px high, not 3px.

**Comment by yaili** [comment]
It would be an accent hr, I mean.

**Comment by barrymcgee** [comment]
@yaili Please propose to the Brochure theme.

**Comment by yaili** [comment]
I don't think there is a use case outside of ubuntu.com yet, so it should only be on this site, no?

**Comment by barrymcgee** [comment]
@yaili As it's already been merged into the Brochure theme and I still think it's right that it's part of the divider pattern, can we wait and see if there's a use case against it? If it then transpires we need to override locally to remove equal height dividers, we can revisit?

**Comment by yaili** [comment]
Are you referring to the accent hr or the vertical dividers always being equal height?

**Comment by barrymcgee** [comment]
This issue - equal height dividers.

**Comment by pmahnke** [comment]
this was fixed with the p-divider__block updates
